tests/ui/type-alias-impl-trait/different_args_considered_equal2.stderr

```diff
@@ -9,11 +9,6 @@ LL | fn get_one<'a>(a: *mut &'a str) -> impl IntoIterator<Item = Opaque<'a>> {
 ...
 LL |         None::<Opaque<'static>>
    |         ^^^^^^^^^^^^^^^^^^^^^^^
-   |
-help: to declare that `impl IntoIterator<Item = Opaque<'a>>` captures `'a`, you can add an explicit `'a` lifetime bound
```

Comment:
This suggestion was wrong -- we were suggesting to add the `+ 'a` to the wrong opaque -- one that *already* captured `'a`.